installer/templates/phx_live/live/home_live.ex

```diff
@@ -0,0 +1,31 @@
+defmodule <%= web_namespace %>.HomeLive do
+  use <%= web_namespace %>, :live_view
+
+  def render(assigns) do
+    <%= web_namespace %>.PageView.render("home.html", assigns)
+  end
+
+  def mount(_params, _session, socket) do
+    {:ok, socket}
+  end
+
+  def handle_params(_params, _uri, socket) do
+    {:noreply, socket}
+  end
+end
+
+defmodule <%= web_namespace %>.HomeLive.DepsCheck do
```

Comment:
I dumped this component here since it's likely not to survive our discussions. If we want to go this direction I'm fine with an additional file in the `live/` dir for a component, but dumping here for experiments :)

Reply:
Right, let's talk about it but I would prefer to keep things as simple as possible and go with more complexity only when people call `mix phx.gen.live`. So for this particular example, I would propose to not use a modal (we can generate it on scaffold like we do with the action fallback), and simply have a div show up when we press the button to check for outdated deps, with no live navigation. If we can have some sort of CSS-based animation for the div showing up, then even better, because we can teach even more things to users.